packages/clone-testing/Cargo.toml

```diff
@@ -11,10 +11,16 @@ description = "Adapter for interacting with cw-multi-test via the cw-orchestrato
 
 [dependencies]
 cosmwasm-std = { workspace = true }
-cw-orch-daemon = { workspace = true }
-cw-orch-core = { workspace = true }
-cw-orch-mock = { workspace = true }
-clone-cw-multi-test = { version = "0.5", features = ["cosmwasm_1_2"] }
+
+# This package depends on published cw-orch version due to the clone-testing-cw-multi-test dependency
+cw-orch-core = { version = "2.0.0" }
+cw-orch-mock = { version = "0.24.0" }
```

Comment:
same question as #480

Reply:
Resolved because this branch will be merged into #480